packages/orbit-components/src/Tabs/components/TabPanels/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+import type { Props } from "./types";
+import { useTabs, TabPanelProvider } from "../../TabContext";
+
+const TabPanels = ({ children, selectedTab, dataTest }: Props) => {
+  const { selected } = useTabs();
+  const isSelected = Boolean(selectedTab || selected);
+
+  return (
+    <div data-test={dataTest}>
+      {React.Children.map(children, (child, idx) => {
+        if (!React.isValidElement(child)) return null;
+        return (
+          (idx === selectedTab || idx === selected) && (
```

Comment:
This relies on the order of the tabs and panels being the same, and unchanged. But imagine one has a disabled Tab and doesn't even has content for the corresponding TabPanel (if it's disabled, why would it even have something there…?). We could rely on id or something like that.

It also mixes the controlled and uncontrolled behaviors. I would expect that in a scenario where I am in full control of the state of the Tab I am also in full control of the TabPanel. I wouldn't expect to be TabPanels handling that, and relying on a Context. Can't we have a possible `active` prop on `TabPanel` that controls whether it is rendered or not? We would remove that responsibility from this `TabPanels` component.